src/Streamer.cpp

```diff
@@ -151,6 +151,13 @@ FileWriter::Streamer::pollAndProcess(FileWriter::DemuxTopic &MessageProcessor) {
     return ProcessMessageResult::OK;
   }
 
+  if (static_cast<std::int64_t>(Message->getTimestamp()) == 0) {
```

Comment:
Cast is not needed here because compare with `0`.

Reply:
:+1: